src/views/settings/CreditsView.tsx

```diff
@@ -18,8 +18,10 @@ export default function CreditsView() {
             <Text style={{ fontSize: 20 }}>
                 <Text style={{ fontWeight: "bold" }}>Nathaniel Kohnen</Text> - Head of Marketing
             </Text>
-            <Text style={{ fontWeight: "700", fontSize: 25, paddingVertical: 10 }}>Sponsors:</Text>
-            <Text style={{ fontSize: 20 }}>Elliot Kovacs</Text>
+            <Text style={{ fontWeight: "700", fontSize: 25, paddingVertical: 10 }}>😎Sponsors:😎</Text>
```

Comment:
Please remove the unnecessary emoji

Reply:
come on bro. Let me have this. I pay your salary

Reply:
This is a professional app, we are not going to destroy it with cring